src/OpenTelemetry/Metrics/DropConfiguration.cs

```diff
@@ -0,0 +1,35 @@
+// <copyright file="DropConfiguration.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+
+namespace OpenTelemetry.Metrics
+{
+    internal class DropConfiguration : MetricStreamConfiguration
+    {
+        private Aggregation aggregation = Aggregation.Drop;
```

Comment:
Unrelated to this PR, but I can't recall the reason why we have `Aggregation.None` and `Aggregation.Drop`.

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#none-aggregation Spec has "None". We thought its easier if we add `Drop` (point to None itself), as its more clear.. Now that I read it more, I think we can simply remove None, and have Drop only.